### PR TITLE
ref(ui): Remove unused `app/icons/*.svg`, `<InlineSvg>`, `svgo-loader`

### PR DESCRIPTION
**Pull request by billyvg**
Also removes associated `webpack` loaders.

We were only using these loaders and `<InlineSvg/>` for this logo: 
image

No need to run these additional loaders for a single svg.

I've checked getsentry for logo/power-features usage (no InlineSvg usage).

### CODE REVIEW

**Comment by billyvg** [comment]
@scefali I don't know how to test Demo mode -- would it be worthwhile to have Demo-mode components in storybook?

**Comment by scefali** [comment]
@billyvg For now, we should probably keep demo mode hidden from storybook (or anything externally facing). If you break something right related to demo mode right now, that's OK. If demo mode ends up sticking around in the long term, we can revisit.

File: src/sentry/static/sentry/app/icons/iconSentryFull.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import SvgIcon from './svgIcon';
+
+type Props = React.ComponentProps<typeof SvgIcon>;
+
+const IconSentryFull = React.forwardRef(function IconSentryFull(
+  props: Props,
+  ref: React.Ref<SVGSVGElement>
+) {
+  return (
+    <svg viewBox="0 0 601 134" width="1em" height="1em" {...props} ref={ref}>
```
Comment (markstory) [inline src/sentry/static/sentry/app/icons/iconSentryFull.tsx:12]: Should the width and height be equal if the image is a rectangle? The old dimensions were 601px wide 134px tall.

**Review by doralchan — APPROVED** [review]
Nice

File: webpack.config.js [hunk 2]
```diff
@@ -255,10 +255,6 @@ let appConfig = {
           },
         },
       },
-      {
-        test: /app\/icons\/.*\.svg$/,
-        use: ['svg-sprite-loader', 'svgo-loader'],
-      },
```
Comment (EvanPurkhiser) [inline webpack.config.js:261]: Damn this is great 🥳

**Pull request merged**